voucher_codes.js

```diff
@@ -64,6 +64,7 @@
         this.count = config.count || 1;
         this.length = config.length || 8;
         this.charset = config.charset || charset("alphanumeric");
+        this.unique_charset = uniqueCharset(this.charset);
```

Comment:
`this.unique_charset = Array.from(new Set(this.charset));` ?

Reply:
we want to have as broad compatibility as possible, so I don't think we can use `Set`